app/models/edm/rights.rb

```diff
@@ -17,12 +17,17 @@ def normalise(string)
       end
 
       def for_api_query(value)
```

Comment:
`#for_api_query` is no longer used anywhere in the app. If that is intentional, please remove it.